docs/tutorials/multimodal/object_detection/inference/detection_inference_quick_start.md

```diff
@@ -0,0 +1,177 @@
+
+Now let's download the dataset
+```python
+download_dir = "./tiny_motorbike_coco"  # specify a target download dir to store this dataset
+
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+data_dir = os.path.join(download_dir, "tiny_motorbike")
+train_path = os.path.join(data_dir, "Annotations", "coco_trainval.json")
+test_path = os.path.join(data_dir, "Annotations", "coco_test.json")
+```
+
+## Creating the `MultiModalPredictor`
+To start, import MultiModalPredictor:
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+### Use a pretrained model
+You can download a pretrained model and construct a predictor with it. 
+In this example, we use the VFNet with ResNext as backbone and Feature Pyramid Network (FPN) as neck.
+
+```python
+checkpoint_name = "vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco"
+num_gpus = 1  # set to -1 to use all GPUs if available
+```
+You can also use other model by setting `checkpoint_name` to other names. 
+Please refer to :ref: `selecting_models` for details about model selection.
+
+As before, we create the MultiModalPredictor with selected checkpoint name and number of GPUs.
+We also need to specify the `problem_type` to `"object_detection"`.
+
+```python
+predictor = MultiModalPredictor(
+    hyperparameters={
+        "model.mmdet_image.checkpoint_name": checkpoint_name,
+        "env.num_gpus": num_gpus,
+    },
+    problem_type="object_detection",
+)
+```
+
+### Use a finetuned model
+You can also use a previously trained/finetuned predictor to run inference with.
+First specify the predictor path, for example:
+```python
+load_path = "./AutogluonModels/ag-20221104_185342"  # replace this with path to your desired predictor
+```
+Then load the predictor:
+```python
+predictor = MultiModalPredictor.load(load_path)
+```
+
+## Setting up data
+
+For COCO format data, we need to provide the path for the data split used for inference.
+
+```python
+test_path = "./tiny_motorbike_coco/tiny_motorbike/Annotations/coco_test.json"
+```
+
+## Running inference
+To run inference, perform:
+
+```python
+pred = predictor.predict(test_path)
+```
+
+To run inference and save results, run the following:
+```python
+pred = predictor.predict(test_path, save_results=True)
+```
+Currently, we convert the results to a pandas `DataFrame` and save into a `.txt` file. 
+The `.txt` file has two columns, `image` and `bboxes`, where
+- in `image`, each row contains the image path
+- in `bboxes`, each row is a list of dictionaries, each one representing a bounding box: 
+  - `{"class": <predicted_class_name>, "bbox": [x1, y1, x2, y2], "score": <confidence_score>}` 
+
+## Reading results
+The returned value `pred` is a `list` and has the following dimensions:
+
+```
+[num_images, num_total_classes, num_detections_per_class, 5]
+```
+
+where 
+- `num_images` is the number of images used to run inference on. 
+- `num_total_classes` is the total number of classes. In this example, `num_total_classes = 20` for VOC dataset
```

Comment:
classes=20?

Reply:
yea for VOC dataset, there is 20 classes in total.